### PR TITLE
fixed so panel title doesn't wrap and 

### LINKED ISSUE #11074: Spark lines are showed below expected position in 5.0.0
After migration from 4.6.0 to 5.0.0 I have noticed weird spark lines position. 
### 5.0.0 ###
screen shot 2018-03-03 at 10 45 18 pm

### 4.6.0 ###
screen shot 2018-03-03 at 10 43 01 pm

I've checked this on Safari as well as Chrome - the result is the same.

### PR DESCRIPTION
**Pull request by Ijin08**
carrot doesn't appear outside panel when panel is thin

fixes #11074

longtitle

### CODE REVIEW

**Review by marefr — CHANGES_REQUESTED** [review]
The panel error message icon is positioned of, see screenshot below:

image

Seems like alerting icons positioning is off now. Verified in chrome, firefox and IE11. See screenshots below.

**Before:**
image

**After:**
image

Besides this it looks good. Also verified that phantomjs rendering still works.

**Comment by Ijin08** [comment]
Latest push:
skarmavbild 2018-06-05 kl 09 24 18

File: public/sass/pages/_dashboard.scss [hunk 1]
```diff
@@ -68,11 +68,17 @@ div.flot-text {
...
+  flex-wrap: nowrap;
+  justify-content: center;
+  padding: 4px 0 4px;
 }
 
 .panel-title-text {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  max-width: calc(100% - 28px);
```
Comment (marefr) [inline public/sass/pages/_dashboard.scss]: Changing to 48px or similar pushes the down arrow icon more to the left and adds some space to the right - I think it looks better - what do you think?

**Review by marefr — CHANGES_REQUESTED** [review]
Have a look at he comment and let me know what you think/push those changes and then we're good to merge this :+1:

**Comment by Ijin08** [comment]
@marefr decreased max-width to -38. I think it is enough, what do you think?
skarmavbild 2018-06-07 kl 13 55 26

**Comment by marefr** [comment]
What, my updated comment disappeared??? I intended to also suggest to add a special rule if alert rule is defined to make more space for the arrow down icon:

```scss
.panel-title-text {
  ..
  max-width: calc(100% - 38px);
  ...

  .panel-has-alert & {
    max-width: calc(100% - 58px);
  }
  ...
}
```

**Review by marefr — APPROVED** [review]
Awesome :heart:

**Pull request merged**